msteams-platform/sbs-tab-with-adaptive-cards.yml

```diff
@@ -0,0 +1,362 @@
+### YamlMime:Tutorial
+title: Teams Tab with Adaptive cards
+metadata:
+  title: Teams Tab with Adaptive Cards
+  description: In this tutorial, you'll learn to set up Adaptive Cards in a tab for Teams.
+  audience: Developer
+  level: Beginner
+  ms.date: 10/21/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: bots/how-to/authentication/auth-aad-sso-bots.md
+  nextTutorialTitle: Read more on bot framework documentation
+  ms.custom: mvc
+  ms.localizationpriority: none
+items:
+- durationInMinutes: 1
+  content: |
+    Adaptive Card tabs are a new way to build tabs in Teams. Instead of embedding web content in an IFrame, you can render Adaptive Cards to a tab. While the front end is rendered with Adaptive Cards, the backend is powered by a bot. The bot is responsible for accepting requests and responding appropriately with the Adaptive Card that is rendered.
+
+    You can build your tabs with relevant user interface (UI) building blocks native on desktop, web, and mobile.
+
+    This article helps you understand:
+    
+    * The changes required to be made to the app manifest.
+    * Identifies how the invoke activity requests and sends information in tab with Adaptive Cards.
+    * Effect on the task module workflow.
+    
+    This step-by-step guide helps you to build tabs with Adaptive Cards in desktop and mobile.
```

Comment:
Please change the resolution of all the screenshots as they are very huge

Reply:
Sure Habiba. Will do that